jmclient/test/test_wallet_rpc.py

```diff
@@ -59,6 +60,16 @@ def setUp(self):
         # start the daemon; note we are using tcp connections
         # to avoid storing certs in the test env.
         # TODO change that.
+        if self.dport is None and self.wss_port is None:
```

Comment:
Curious about the splitting into three cases here; could we not just call `get_free_tcp_ports(2)` before the `if` clauses and then use them according to the case?

Reply:
We could, but does it matter?

Reply:
No, it's just my instinct to not repeat things if unnecessary. But no need to change if you don't want to.

Reply:
Yes, I'm too lazy here, besides could argue about not doiing that. Ok to merge from you?